### PR TITLE
chore(iotevents): rename _bind from bind

### PR DESCRIPTION
**Pull request by yamatatsu**
I apologize for my uncooperative comment in the previous PR.
I've refactored it!

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by mrgrain** [comment]
Hey @yamatatsu thanks for the contribution. I'm feeling a bit lost here.

What's the background to this? Did you have someone specific from the team in mind who should take a look here?

File: packages/@aws-cdk/aws-iotevents/lib/action.ts [hunk 1]
```diff
@@ -18,8 +18,9 @@ export interface ActionBindOptions {
 export interface IAction {
   /**
    * Returns the AWS IoT Events action specification.
+   * @internal
    */
-  bind(scope: Construct, options: ActionBindOptions): ActionConfig;
+  _bind(scope: Construct, options: ActionBindOptions): ActionConfig;
```
Comment (TheRealAmazonKendra) [inline packages/@aws-cdk/aws-iotevents/lib/action.ts:23]:
```suggestion
  public _bind(scope: Construct, options: ActionBindOptions): ActionConfig;
```

**Review by TheRealAmazonKendra — CHANGES_REQUESTED** [review]
@yamatatsu No need to apologize! Thanks for picking this up even though you certainly didn't have to. @mrgrain  I was being a bit grumpy that we hadn't requested a change on an interface when it was originally implemented and commented on that. It's our own convention that  we weren't following. It was out of scope for that PR, though. 

Just one place below where `public` wasn't set (suggested change inline).

File: packages/@aws-cdk/aws-iotevents/lib/action.ts [hunk 2]
```diff
@@ -18,8 +18,9 @@ export interface ActionBindOptions {
 export interface IAction {
   /**
    * Returns the AWS IoT Events action specification.
+   * @internal
    */
-  bind(scope: Construct, options: ActionBindOptions): ActionConfig;
+  public _bind(scope: Construct, options: ActionBindOptions): ActionConfig;
```
Comment (TheRealAmazonKendra) [inline packages/@aws-cdk/aws-iotevents/lib/action.ts:23]:
```suggestion
  _bind(scope: Construct, options: ActionBindOptions): ActionConfig;
```

**Review by TheRealAmazonKendra — DISMISSED** [review]

**Comment by TheRealAmazonKendra** [comment]
Oops. I was wrong.

**Review by TheRealAmazonKendra — APPROVED** [review]

**Pull request merged**